packit_service/worker/parser.py

```diff
@@ -702,6 +702,10 @@ def parse_merge_request_comment_event(
             logger.warning("No Gitlab username from event.")
             return None
 
+        if actor in {"packit-as-a-service", "packit-as-a-service-stg"}:
```

Comment:
Please add a TODO to make it part of the git forge config.

Reply:
discussed in person, the problem here is we want to ignore comments by both instances, not only for the one that is processing the message